Core/Model/Base/ModelCore.php

```diff
@@ -129,20 +129,23 @@ abstract public static function tableName(): string;
      */
     public function __construct(array $data = [])
     {
-        if (self::$dataBase === null) {
+        if (!(self::$dataBase instanceof DataBase)) {
             self::$dataBase = new DataBase();
-
-            $tables = Cache::get('fs_checked_tables');
-            if (is_array($tables) && !empty($tables)) {
-                self::$checkedTables = $tables;
-            }
         }
 
-        if (static::tableName() !== '' && false === in_array(static::tableName(), self::$checkedTables, false) && $this->checkTable()) {
-            $this->toolBox()->i18nLog()->debug('table-checked', ['%tableName%' => static::tableName()]);
-            self::$checkedTables[] = static::tableName();
-            Cache::set('fs_checked_tables', self::$checkedTables);
-        }
+        DatabaseUpdater::check(static::tableName());
```

Comment:
Aquí el asunto es llamar a la función checkTable() del modelo solamente si la tabla no está en la lista de comprobada. Algo así:

if (false === DatabaseUpdater::checked(static::tableName()) {
   $this->checkTable()
}

Reply:
hecho!